### DISCUSSION TITLE
How to hide row number column in Table view?

### DISCUSSION THREAD

**Discussion by sundeer** [marcusolsson/obsidian-projects#741] [Help]
If I show more than 8 fields in the table view, the row number column number header disappears. This shifts all the other column headers to the left, skewing the rest of the column borders.

![Screenshot 2023-11-21 at 10 52 21 AM](https://github.com/marcusolsson/obsidian-projects/assets/2971405/17c449fb-1893-456a-a311-3a1dc6c2fade)

![Screenshot 2023-11-21 at 10 52 34 AM](https://github.com/marcusolsson/obsidian-projects/assets/2971405/454f02ff-6a42-4429-aded-29f28f0c4418)

I need to either hide the row number column or figure out why this is happening.

**Comment by Acylation** [comment]
Hi I can see the problem clearly from your screenshots. Thanks for your report!

I may triage this as a bug because the Table view should occupy the hole horizontal region. It seems that you're using a wider screen? Could you provide more info related to your devices, like operating system and screen size? A screenshot of the entire Obsidian app would be quite appreciated if you can put an anonymous one.

**Reply by sundeer** [reply]
No prob
![Screenshot 2023-11-21 at 2 20 21 PM copy](https://github.com/marcusolsson/obsidian-projects/assets/2971405/07cf6a7e-78c1-45c2-9276-a6e9cb82af8c)
![Screenshot 2023-11-21 at 2 20 43 PM](https://github.com/marcusolsson/obsidian-projects/assets/2971405/174d172e-78aa-4a64-aace-74d4942ddcf7)


Here are the plugins I'm using.
```
drwxr-xr-x  5 rs  staff   160B Nov 15 11:12 buttons
drwxr-xr-x  6 rs  staff   192B Nov 15 11:12 dataview
drwxr-xr-x  6 rs  staff   192B Nov 17 15:07 dbfolder
drwxr-xr-x  7 rs  staff   224B Nov 15 11:13 image-window
drwxr-xr-x  6 rs  staff   192B Nov 15 11:12 initiative-tracker
drwxr-xr-x  5 rs  staff   160B Nov 17 09:33 multi-column-markdown
drwxr-xr-x@ 6 rs  staff   192B Nov 21 10:25 notion-like-tables
drwxr-xr-x  6 rs  staff   192B Nov 21 14:04 obsidian-5e-statblocks
drwxr-xr-x  6 rs  staff   192B Nov 15 11:12 obsidian-admonition
drwxr-xr-x  4 rs  staff   128B Nov 15 11:11 obsidian-advanced-uri
drwxr-xr-x  5 rs  staff   160B Nov 15 11:08 obsidian-columns
drwxr-xr-x  5 rs  staff   160B Nov 15 11:12 obsidian-dice-roller
drwxr-xr-x  7 rs  staff   224B Nov 15 11:11 obsidian-dictionary-plugin
drwxr-xr-x  6 rs  staff   192B Nov 15 11:13 obsidian-excalidraw-plugin
drwxr-xr-x  6 rs  staff   192B Nov 15 11:12 obsidian-icon-folder
drwxr-xr-x  5 rs  staff   160B Nov 15 11:12 obsidian-kanban
drwxr-xr-x  6 rs  staff   192B Nov 15 11:12 obsidian-leaflet-plugin
drwxr-xr-x  6 rs  staff   192B Nov 15 11:12 obsidian-map-view
drwxr-xr-x  6 rs  staff   192B Nov 15 11:12 obsidian-meta-bind-plugin
drwxr-xr-x  6 rs  staff   192B Nov 18 14:26 obsidian-projects
drwxr-xr-x  6 rs  staff   192B Nov 15 11:11 obsidian-style-settings
drwxr-xr-x  6 rs  staff   192B Nov 15 11:13 obsidian-textgenerator-plugin
drwxr-xr-x  6 rs  staff   192B Nov 15 11:09 obsidian-view-mode-by-frontmatter
drwxr-xr-x  4 rs  staff   128B Nov 15 11:11 obsidian-vimrc-support
drwxr-xr-x  6 rs  staff   192B Nov 15 11:13 obsidian-wordnet-plugin
drwxr-xr-x  6 rs  staff   192B Nov 15 11:12 quickadd
drwxr-xr-x  6 rs  staff   192B Nov 15 11:12 table-editor-obsidian
drwxr-xr-x  6 rs  staff   192B Nov 15 11:11 templater-obsidian
```
